tests/model_hub_tests/pytorch/test_llm.py

```diff
@@ -222,10 +217,15 @@ def test_convert_model_nightly(self, name, type, ie_device):
         ("baichuan", "baichuan-inc/Baichuan2-7B-Base"),
         pytest.param("chatglm", "THUDM/chatglm3-6b",
                      marks=pytest.mark.xfail(reason="Accuracy validation failed")),
+        ("falcon", "tiiuae/falcon-7b-instruct"),
         ("fuyu", "ybelkada/fuyu-8b-sharded"),
+        ("gemma", "beomi/gemma-ko-7b"),
```

Comment:
do we really need 7b model while there 2b?

Reply:
Good point, will change in next PR